spec.bs

```diff
@@ -786,9 +786,10 @@ An interest group is a [=struct=] with the following items:
 : <dfn>expiry</dfn>
 :: A point in time at which the browser will forget about this interest group.
 : <dfn>owner</dfn>
-:: An [=url/origin=].
+:: An [=url/origin=]. Frames that join interest groups owned by `owner` must either be served from origin `owner`,
+  or another origin delegated by `owner`. The scheme must be https.

```

Comment:
What mechanism defines the delegation? Can we point to that here to make it more clear?

Reply:
I think it's the .well-known stuff. There's an open [PR](https://github.com/WICG/turtledove/pull/508) for it.

Reply:
I'll add a TODO to update this to link to the delegation section.